atc/wrappa/security_handler.go

```diff
@@ -18,6 +18,6 @@ func (handler SecurityHandler) ServeHTTP(w http.ResponseWriter, r *http.Request)
 	w.Header().Set("X-XSS-Protection", "1; mode=block")
 	w.Header().Set("X-Content-Type-Options", "nosniff")
 	w.Header().Set("X-Download-Options", "noopen")
-	w.Header().Set("Cache-Control", "no-store, private")
```

Comment:
Actually 😁 could we add `no-store` back here? It does seem like a reasonable thing for API responses, assuming my understanding is correct, and I'm guessing `private` alone won't satisfy the vulnerability scanner if it still stores responses to disk.

Sorry about that 😬  I think I felt that if it was difficult overriding the `Cache-Control` header per route, then we'd be better off still allowing assets to be properly cached - but given that it's easy to cache assets while still preventing API responses from being cached, I think `no-store` is reasonable to keep

Reply:
Good point! I'll add it back in.